AuthorDepositForm.inc.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/**
+* @file plugins/generic/sword/AuthorDepositForm.inc.php
+*
+* Copyright (c) 2003-2018 Simon Fraser University
+* Copyright (c) 2003-2018 John Willinsky
+* Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+*
+* @class AuthorDepositForm
+* @ingroup plugins_generic_sword
+*
+* @brief Form to perform an author's SWORD deposit(s)
+*/
+
+import('lib.pkp.classes.form.Form');
+
+class AuthorDepositForm extends Form {
+	/** @var $_context Context */
+	protected $_context = null;
+
+	/** @var $_plugin SwordPlugin */
+	protected $_plugin = null;
+
+	/** @var $_submission Submission */
+	protected $_submission = null;
+
+	/**
+	 * Constructor
+	 * @param $plugin SwordPlugin
+	 * @param $context Context
+	 * @param $submission Submission
+	 */
+	public function __construct(SwordPlugin $plugin, Context $context, Submission $submission) {
+		$this->_plugin = $plugin;
+		$this->_context = $context;
+		$this->_submission = $submission;
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_USER);
+		parent::__construct($plugin->getTemplatePath() . 'authorDepositForm.tpl');
+	}
+
+	/**
+	 * Get reference to the sword plugin
+	 * @return SwordPlugin
+	 */
+	public function getSwordPlugin() {
+		return $this->_plugin;
+	}
+
+	/**
+	 * Initialize plugin settings form
+	 *
+	 * @return void
+	 */
+	public function initData() {
```

Comment:
Is there a reason this function is being overridden with an empty implementation? If not, I'd suggest removing it.